source/reference/minio-mc/mc-replicate-add.rst

```diff
@@ -100,13 +104,24 @@ Parameters
    :required:
 
    Specify the credentials, destination deployment, and bucket of the remote location. 
+   Value may be an alias/bucket, location based (IP or URL), or path based.
 
    For example, a URL based target might look like the following:
 
    .. code-block::
 
       https://user:secret@myminio.cloudprovider.tld:9001/bucket
 
+   .. versionchanged:: mc RELEASE.2024-03-03T00-13-08Z
+
+      The ``--remote-bucket`` value may be an :ref:`alias`.
```

Comment:
```suggestion
      The ``--remote-bucket`` supports specifying an existing :ref:`alias <alias>`
```

Not sure if that's the right tag, i think we created a ref somewhere.

Reply:
It's correct. fixed